scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -119,10 +150,16 @@
     <span class="badge badge-dark">MITOMAP</span>
     <br>
   {% endif %}
+  </div>
```

Comment:
Ah, but then the tooltip will not be on anything really if there is no badge in the cell? Before there was at the least an `freq else: ~`. I tested and could not trigger the tooltip for for rare or cancer variants without badges on Safari at least?

Reply:
Good point. I have divided the information over the pop freq column and observed column